### ISSUE TITLE
1.19 - 1.19.3

### ISSUE THREAD

**Issue by gaycookie** [gaycookie/Transparent-Cosmetics#10]
I find it quite intriguing that I have yet to comprehend the mindset of most mod developers, and I doubt I ever will. Why would someone choose to remain two versions behind the latest release and stubbornly remain in that position?

Nevertheless, as I mentioned earlier in the update, I intend to explore the other three or four versions as well. However, my primary objective was to ensure the functionality of version 1.19.4, as I personally believe that the most recent version holds greater significance especially in the long-term.

I'll work through the versions in sequential order, starting from 1.19 and progressing to 1.19.3.

**Comment by xR4YM0ND** [comment]
I've spoken to some mod devs and a lot of them said they gonna skip from 1.19.2 to 1.20

I think 1.19.2 will be like 1.18.2 in long term, but that's just my prediction.

**Comment by douglasparker** [comment]
> I've spoken to some mod devs and a lot of them said they gonna skip from 1.19.2 to 1.20
> 
> I think 1.19.2 will be like 1.18.2 in long term, but that's just my prediction.

This is the same conclusion that I have come to as well.

My best guess is because there are often breaking changes in point releases `1.19.x` that are not backwards compatible and that mod developers tend to settle on supporting a certain version above others.

**Comment by gaycookie** [comment]
Yea so, code-wise quite some has changed within Minecraft with at least 1.19.2 and 1.19.4.
I've not checked 1.19.3 yet, but that is mainly because I made a mess of this repository and also cleaning that now.
And that is already my simple ass mod, I can't imagine what happen for other bigger mods. 😰

**Comment by douglasparker** [comment]
> Yea so, code-wise quite some has changed within Minecraft with at least 1.19.2 and 1.19.4.
> I've not checked 1.19.3 yet, but that is mainly because I made a mess of this repository and also cleaning that now.
> And that is already my simple ass mod, I can't imagine what happen for other bigger mods. 😰

Yeah, there are a LOT of mods out there and getting them all to do major updates for each point release seems to take a really long time.

This is probably why they all primarily pick a version to settle on, and then the community tends to prefer playing on that version because of all the mods support.

I was looking into playing on 1.19.4, and while there were a lot of mods ported to that version, there were also a lot of mods that are skipping that version as well. So we ended up on 1.19.2.

**Comment by gaycookie** [comment]
Yea I can imagine, I'm a little salty over what Mojang/Microsoft did here haha!
But with a little bit of luck will I have three versions one for 1.19-1.19.2, 1.19.3 and 1.19.4.
So to control the damage a little bit on my side.

**Comment by gaycookie** [comment]
So.. I hope I've done it now! 1.19 to 1.19.4! [Release 1.5.1](https://github.com/gaycookie/Transparent-Cosmetics/releases/1.5.1)
I would love to see some feedback if the mods work for you guys!

![giphy](https://github.com/gaycookie/Transparent-Cosmetics/assets/34284353/8761759f-50b1-44ca-8aa9-dfeba238282e)

**Comment by ghost** [comment]
> So.. I hope I've done it now! 1.19 to 1.19.4! [Release 1.5.1](https://github.com/gaycookie/Transparent-Cosmetics/releases/1.5.1)
> I would love to see some feedback if the mods work for you guys!
> 
> ![giphy](https://github.com/gaycookie/Transparent-Cosmetics/assets/34284353/8761759f-50b1-44ca-8aa9-dfeba238282e)

Awesome! I will test it tonight if you can publish a modrinth release!

**Comment by gaycookie** [comment]
Already done! =D
